stack.yaml

```diff
@@ -3,11 +3,11 @@ packages:
   - .
 extra-deps:
   - ghc-lib-parser-8.10.1.20200523
-  - ghc-lib-parser-ex-8.10.0.11
+  - ghc-lib-parser-ex-8.10.0.13
 # To test hlint against experimental builds of ghc-lib-parser-ex,
 # modify extra-deps like this:
-#  - archive: /users/shaynefletcher/project/ghc-lib-parser-ex.git/ghc-lib-parser-ex-8.10.0.11.tar.gz
-#    sha256: "0000000000000000000000000000000000000000000000000000000000000000"
+#  - archive: /users/shaynefletcher/project/ghc-lib-parser-ex.git/ghc-lib-parser-ex-8.10.0.14.tar.gz
```

Comment:
The 14 and the 13 above aren't matched.

Reply:
That's ok, that should read 14 not that it's important.